wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -205,6 +218,22 @@ final class SelfProfileViewController: UIViewController {
 
     // MARK: - Events
 
+    private func onTeamCreationBannerInteraction(_ action: SelfProfileViewCallToActionBanner.Action) {
+        switch action {
+        case .createWireTeam:
+            userDidTapCreateTeam()
+        }
+    }
+
+    private func userDidTapCreateTeam() {
+        // TODO: [WPB-10347] Present team creation flow
+//        let vc = IndividualToTeamMigrationViewController(
+//            features: ,
+//            useCase: ,
+//        )
+//        present(vc, animated: true)
+    }
```

Comment:
can we do this now? otherwise it should be the ticket number of a followup PR instead of the current one